lib/sparql_to_sw_solr/physical_field.rb

```diff
@@ -0,0 +1,51 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module PhysicalField
+
+      private
+
+      def physical_values
+        physical_solutions.each_solution do |soln|
+          next unless soln.bindings.keys.include?(:noteType)
+          @note_type = soln.noteType.to_s
+          @physical1 = soln.extentLabel_1.to_s
+          @physical3 = soln.instanceDimensions_3.to_s
+          note_case(soln)
```

Comment:
Darren and I paired, and we changed the method to get the first value from the query solution, even though the way that the sparql query returned results, the value would always be the same for `@physical1` and `@physical3` for each query solution.